frontend/src/data/tasks.json

```diff
@@ -344,7 +344,7 @@
-        "descriptionFr": "Vous n'avez pas besoin d'arrêter de travailler pour toucher votre pension de retraite du RPC. Vous pouvez commencer à percevoir votre pension du RPC entre 60 et 70 ans. Vous pouvez déposer votre demande dès l'âge de 59 ans, quel que soit l'âge auquel vous souhaitez commencer à percevoir votre pension. Vous devez présenter une demande lorsque vous êtes certain(e) de la date à laquelle vous souhaitez commencer.\n\nLe RPC vous permet de choisir quand débuter votre pension de retraite, et chaque mois que vous reportez augmente votre montant mensuel. Le montant mensuel le plus élevé que vous pouvez recevoir arrive à 70 ans, après quoi il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à toucher votre rente dès l'âge de 60 ans, mais avec une réduction permanente.\n\nLors de votre demande, vous devrez choisir la date à laquelle vous souhaitez commencer à percevoir votre pension.\n- Dès que vous remplissez les conditions requises (le mois suivant votre 60 anniversaire)\n- À une date précise que vous choisissez (après l'âge de 60 ans)\n\nVous pouvez postuler sur papier en personne ou par courrier. Pour une réponse plus rapide, postulez en ligne. Pour plus de détails, voir les liens ci-dessous.",
+        "descriptionFr": "Vous n'avez pas besoin d'arrêter de travailler pour recevoir votre pension du RPC. Vous pouvez commencer à recevoir votre pension du RPC entre 60 et 70 ans. Vous pouvez déposer votre demande dès l'âge de 59 ans, quel que soit l'âge auquel vous souhaitez commencer à recevoir votre pension. Vous devez présenter une demande lorsque vous êtes certain(e) de la date à laquelle vous souhaitez commencer.\n\nLe RPC vous permet de choisir quand commencer votre pension de retraite, et chaque mois que vous reportez, le montant mensuel augmente. Le montant le plus élevé à recevoir est à l'âge de 70 ans. Après cet âge, il n'y a aucun avantage à attendre. Si vous avez besoin d'argent plus tôt, vous pouvez commencer à recevoir votre pension dès l'âge de 60 ans, mais avec une réduction permanente.\n\nLors de votre demande, vous devrez choisir la date à laquelle vous souhaitez commencer à recevoir votre pension.\n- Dès que vous remplissez les conditions requises (le mois suivant votre 60 anniversaire)\n- À une date précise que vous choisissez (après l'âge de 60 ans)\n\nVous pouvez présenter votre demande en format papier, en personne ou par la poste. Pour une réponse plus rapide, faites votre demande en ligne. Pour plus de détails, consultez ces liens.",
```

Comment:
The list items in this block do not have semicolons and a period like the others do. Was this intentional?